src/codegen/codegen_c_visitor.cpp

```diff
@@ -1627,11 +1627,11 @@ void CodegenCVisitor::visit_eigen_newton_solver_block(ast::EigenNewtonSolverBloc
     printer->add_newline();
     auto float_type = default_float_data_type();
     int N = node->get_n_state_vars()->get_value();
-    printer->add_line("Eigen::Matrix<{}, {}, 1> X;"_format(float_type, N));
+    printer->add_line("Eigen::Matrix<{}, {}, 1> X_operator;"_format(float_type, N));
```

Comment:
I know I'm nitpicking here.. but why did you decide to call it "X_operator"? it's not really an operator, is it?